plugins/00_dokku-standard/commands

```diff
@@ -34,6 +34,15 @@ case "$1" in
         ;;
 
       dockerfile)
+        # extract first port from Dockerfile
+        DOCKERFILE_PORT=$(grep EXPOSE Dockerfile | head -1 | awk '{ print $2 }')
+        if [[ -n "$DOCKERFILE_PORT" ]];then
+          if [[ -f "$DOKKU_ROOT/$APP/ENV" ]] && [[ "$(grep DOKKU_DOCKERFILE_PORT $DOKKU_ROOT/$APP/ENV) > /dev/null 2>&1; echo $?" == "0" ]];then
+            sed  -i -e "s:DOKKU_DOCKERFILE_PORT=.*:DOKKU_DOCKERFILE_PORT=$DOCKERFILE_PORT:g" "$DOKKU_ROOT/$APP/ENV"
+          else
+            echo "export DOKKU_DOCKERFILE_PORT=$DOCKERFILE_PORT" >> "$DOKKU_ROOT/$APP/ENV"
```

Comment:
This is the sort of thing I wanted to avoid. If we created a flag for config:set to not rebuild, that would be ideal, as there are other plugins (like databases) that could set variables during pre-deploy. Thoughts?